libvast/native-plugins/summarize.cpp

```diff
@@ -232,7 +232,7 @@ struct group_by_column {
            schema_rt.resolve_key_suffix(extractor, schema.name())) {
         auto field = schema_rt.field(offset);
         auto& column = result.emplace_back();
-        column.input = std::move(offset);
+        column.input = offset;
```

Comment:
NB: we really should have use-after-move clangd check as part of our dev environment, as provided by clang-tidy.

Reply:
There's one for clang-tidy, but it's not working apparently. At least for me it did not show up here.

https://releases.llvm.org/5.0.1/tools/clang/tools/extra/docs/clang-tidy/checks/misc-use-after-move.html#misc-use-after-move